docs/languages/scala.md

```diff
@@ -35,7 +35,12 @@ M.config = function()
     excludedPackages = {},
   }
   metals_config.init_options.statusBarProvider = false
-  require("metals").initialize_or_attach { metals_config }
+  require("metals").initialize_or_attach(metals_config)
+  vim.api.nvim_create_autocmd({ "BufEnter", "BufWinEnter" }, {
+    pattern = { "*.scala", "*.sbt", "*.sc" },
+    callback = function() require("metals").initialize_or_attach(config) end,
```

Comment:
Previously, this was:
```
initialize_or_attach { config }
```
and hence the config is not valid. Metals will replace an invalid config with a default config, therefore, our previous setup seems to run normally.

Reply:
Another plus is metals' `initialize_or_attach` function is only called when the current buffer is scala.